.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
     - checkout
     - run: make promu
-    - run: make check_license style vet unused staticcheck test build
+    - run: make check_license style vet unused staticcheck test-short build
```

Comment:
We should be able to run long tests in CIrcleCI if we add `- setup_remote_docker` above this line.